examples/src/main/scala/zio/dynamodb/examples/dynamodblocal/StudentZioDynamoDbExample.scala

```diff
@@ -45,29 +43,30 @@ object StudentZioDynamoDbExample extends App {
   )
 
   private val dynamoDbLayer: ZLayer[Any, Throwable, DynamoDb] =
-    (http4s.default ++ awsConfig) >>> config.configured() >>> dynamodb.customized { builder =>
-      builder.endpointOverride(URI.create("http://localhost:8000")).region(Region.US_EAST_1)
+    (netty.NettyHttpClient.default ++ awsConfig) >>> config.AwsConfig.default >>> dynamodb.DynamoDb.customized {
```

Comment:
Replaced http4s with netty in all tests because if the db is not running http4s hangs while netty throws an error.

Reply:
is this worthwhile  reporting back to Daniel ?

Reply:
@vigoo - just fyi